src/metrics/case_growth.ts

```diff
@@ -0,0 +1,49 @@
+import { COLOR_MAP } from 'enums/interventions';
+import { Level, Levels } from 'enums/levels';
+
```

Comment:
I kinda' feel like this `src/metrics/` folder should be a subfolder of a new `src/ui` (or `src/content` or `src/constants`, w/e) folder (basically a broader folder that's meant to hold all of our constants for text, colors, UI thresholds, etc.)

Reply:
cool they are all in common